src/dlgTriggerEditor.cpp

```diff
@@ -4757,42 +4763,63 @@ void dlgTriggerEditor::slot_set_pattern_type_color(int type)
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 1:
         palette.setColor(QPalette::Text, QColor(Qt::blue));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 2:
         palette.setColor(QPalette::Text, QColor(195, 0, 0));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 3:
         palette.setColor(QPalette::Text, QColor(0, 195, 0));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 4:
         palette.setColor(QPalette::Text, QColor(0, 155, 155));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 5:
         palette.setColor(QPalette::Text, QColor(137, 0, 205));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 6:
         palette.setColor(QPalette::Text, QColor(100, 100, 100));
         pItem->lineEdit_pattern->hide();
         pItem->pushButton_fgColor->show();
         pItem->pushButton_bgColor->show();
+        pItem->pushButton_prompt->hide();
+        break;
+    case 7:
+        palette.setColor(QPalette::Text, QColor(Qt::black));
+        pItem->lineEdit_pattern->hide();
+        pItem->pushButton_fgColor->hide();
+        pItem->pushButton_bgColor->hide();
+        if (mpHost->mTelnet.mGA_Driver) {
+            pItem->pushButton_prompt->setText(tr("match on the prompt line"));
+            pItem->pushButton_prompt->setToolTip(QString());
+        } else {
+            pItem->pushButton_prompt->setText(tr("match on the prompt line (disabled)"));
+            pItem->pushButton_prompt->setToolTip(tr("A Go-Ahead (GA) signal from the game is required to make this feature work"));
```

Comment:
This is a sentence - so should it not end in a `.`? OTOH I guess I can fix that when I merge it into # 1334 and turn the tool-tip into RichText/HTML like all the others i.e. it will be revised to:

     pItem->pushButton_prompt->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
                                          .arg(tr("<p>A <b>Go-Ahead</b> (GA) signal from the game is required to make this feature work.</p>"));

Reply:
It should not in a tooltip. See Firefox for example:

![workspace 1_238](https://user-images.githubusercontent.com/110988/32311242-c9e3d302-bf97-11e7-98a0-1ac25eca3e73.png)

Reply:
Ah,  I've used a bit of DuckDuckGoFu (like GoogleFu, but it doesn't track where you go and use that to bias future search results) - and found that I've been straying into what Micro$oft calls _info-tips_ territory:

* A  MS [Tooltips and Infotips](https://msdn.microsoft.com/en-us/library/windows/desktop/dn742443(v=vs.85).aspx) Design Guidlines document.
* An item from "The General Mission Analysis Tool" wiki [Writing Tooltips](http://gmatcentral.org/display/GW/Writing+Tooltips) (GMAT is an open-source space mission analysis tool).
* A topic from the User Experience Stack Exchange site [To tooltip or not to tooltip?](https://ux.stackexchange.com/q/1245).

In this particular case the _tip_ gives significant information (the trigger won't work if the MUD Server doesn't use GA) which suggests it is an _info_ (long) rather than a _tool_ (short) tip but it needs further documentation so that the user does not have to access the tip in order to use the control effectively...

Perhaps the button text needs revising: "match on prompt lines (disabled, Server with GA required)"?

Reply:
I'm happy with the text and tooltip combination I've got right now, where the _why_ is left to the tooltip. I don't want to say why is it disabled upfront because while it's useful the first time you see it, it won't be that interesting for the hundredth time and it can stay hidden in a tooltip.

Reply:
OK.